### PR TITLE
added init db config

### PR DESCRIPTION
**Pull request by samodovdi**

### CODE REVIEW

File: template/src/ServiceName.Common/Migrations/20201124200313_Init.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
```
Comment (samodovdi) [inline template/src/ServiceName.Common/Migrations/20201124200313_Init.cs:1]: do we need to add Init migration to the template?

**Review by NovichikhinAlexey — APPROVED** [review]

File: template/src/ServiceName.Common/Persistence/Repositories/ExampleRepository.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            }
+            catch (DbUpdateException e) when (e.InnerException is PostgresException pgEx && pgEx.SqlState == "23505")
+            {
+                context.Examples.Update(example);
+
+                await context.SaveChangesAsync();
+            }
+        }
+
+        public async Task<ExampleEntity> GetAsync(string id)
```
Comment (KonstantinRyazantsev) [inline template/src/ServiceName.Common/Persistence/Repositories/ExampleRepository.cs:36]: To be exact, this is GetOrDefault. `FindAsync` returns null if the record is not found

File: template/src/ServiceName.Worker/Startup.cs [hunk 3]
```diff
@@ -1,32 +1,47 @@
...
+using ServiceName.Common.Persistence.DbContexts;
 using ServiceName.Worker.Modules;
+using Swisschain.Extensions.EfCore;
 using Swisschain.Sdk.Server.Common;
 
 namespace ServiceName.Worker
 {
     public sealed class Startup : SwisschainStartup<AppConfig>
     {
+        public AppConfig Config { get; }
```
Comment (KonstantinRyazantsev) [inline template/src/ServiceName.Worker/Startup.cs:19]: There is `Config` property already defined in SwisschainStartup

File: template/src/ServiceName.Worker/Startup.cs [hunk 4]
```diff
@@ -1,32 +1,47 @@
...
+
         public Startup(IConfiguration configuration)
             : base(configuration)
         {
+            Config = configuration.Get<AppConfig>();
```
Comment (KonstantinRyazantsev) [inline template/src/ServiceName.Worker/Startup.cs:24]: This is already done in the base class

**Review by KonstantinRyazantsev — APPROVED** [review]

**Pull request merged**